pkg/apis/config/v1alpha1/mesh_config.go

```diff
@@ -106,6 +110,29 @@ type TracingSpec struct {
 	Endpoint string `json:"endpoint,omitempty"`
 }
 
+// ExternalAuthorizationSpec is a type to represent external authorization configuration
+type ExternalAuthzSpec struct {
```

Comment:
Let's make this struct specific to OPA? Don't think this needs to be made generic just in its name.